config.yaml

```diff
@@ -1,4 +1,15 @@
 endpoints:
+  - name: ssh
+    group: core
+    url: "ssh://127.0.0.1"
+    ssh:
+      user: "root"
+      password: "password"
+      command: "uptime"
+    interval: 1m
+    conditions:
+      - "[CONNECTED] == true"
+      - "[EXIT_CODE] == 0"
```

Comment:
```suggestion
```
The default configuration should stay as minimal as possible & work out of the box.

If you'd like to create a more in-depth example for endpoints of type SSH, you could always add an example in the [.examples](https://github.com/TwiN/gatus/tree/master/.examples) folder